pages/design/atomic-template-empty.md

```diff
@@ -34,4 +33,42 @@ There are no major accessibility support issues known for this rule.
 
 ## Background
 
-- 
\ No newline at end of file
+- 
+
+## Test Cases
+
```

Comment:
I think, there needs to be a clear heading for all each of the test type groups, namely `Passed`, `Failed` and `Inapplicable`.

@annethyme seems to agree on this as well.

I understand the need to have a heading for each of the test cases like `pass example 1`, but this to me is redundant information, would rather use the short description that we aim to address via https://github.com/auto-wcag/auto-wcag/issues/177 as a heading?

Also for automatic exporting of test cases as per this PR - https://github.com/auto-wcag/auto-wcag/pull/213 there needs to be a way to decipher which group or test type a code snippet belongs to, I use the group heading to discern if a test case is passed, failed of inapplicable, see https://github.com/auto-wcag/auto-wcag/pull/213/files#diff-b0a4bd50b949f5f4482b61265a629b5aR99

I suggest we keep the group headings for test cases.